### ISSUE TITLE
False positive when using unpacked tuple parameter type in function signature

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#3971]
The following code should type check without error.

```python
def func(x: Callable[[Unpack[tuple[int, ...]]], None]):
    x() # Expected 1 more positional argument
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.272. It will also be included in a future release of pylance.
